src/Pmi/Controller/DefaultController.php

```diff
@@ -447,15 +447,15 @@ public function participantAction($id, Application $app, Request $request)
             $patientStatusForm = $patientStatus->getForm($isCommentRequired);
             $patientStatusForm->handleRequest($request);
             if ($patientStatusForm->isSubmitted()) {
-                $patientStatusData = $app['em']->getRepository('patient_status')->fetchOneBy([
+                $patientStatusData = $app['em']->getRepository('patient_status')->fetchBy([
                     'participant_id' => $id,
                     'organization' => $app->getSiteOrganizationId()
-                ]);
+                ], ['history_id' => 'desc'], 1);
                 if (!empty($patientStatusData) && empty($patientStatusForm['comments']->getData())) {
                     $patientStatusForm['comments']->addError(new FormError('Please enter comment'));
                 }
                 if ($patientStatusForm->isValid()) {
-                    $patientStatusId = !empty($patientStatusData) ? $patientStatusData['id'] : null;
+                    $patientStatusId = !empty($patientStatusData) ? $patientStatusData[0]['id'] : null;
```

Comment:
Now that we're adding the unique constraint and cleaning up the data, do we still need these changes?

Reply:
Yes, we don't need them, I'll update. Thanks!